appeals/web/src/server/lib/graph-request.js

```diff
@@ -3,15 +3,23 @@ import config from '@pins/appeals.web/environment/config.js';
 import got from 'got';
 import pino from './logger.js';
 
-const [requestLogger, responseLogger] = createHttpLoggerHooks(pino, config.logLevelStdOut);
+const [requestLogger, responseLogger, retryLogger] = createHttpLoggerHooks(
+	pino,
+	config.logLevelStdOut
+);
 
 export const prefixUrl = 'https://graph.microsoft.com/v1.0/';
 
 const instance = got.extend({
 	prefixUrl,
+	retry: {
+		limit: 3,
+		statusCodes: [500, 502, 503, 504]
+	},
```

Comment:
this is it for what concerns GOT. A parameter in the extend method.
We could work for storing this value somewhere common and easy to change.
I'd love some advice, @beejjacobs

Reply:
also turns out that, all over the code, we have many slightly different implementation of got.
We could, one day, refactor everything as a package.

@francescobrenci-AR @Maby0

Reply:
This is a separate instance of got that is used to fetch Active Directory user info through MS Graph...